subschemas/substrate-api-proxy/src/resolvers/Query/democracy.ts

```diff
@@ -1,12 +1,13 @@
 import { Context } from '../../types';
 import { getCallParams } from '../../utils/call';
 import { notEmpty } from '../../utils/notEmpty';
+import type {DemocracySummary, Democracy} from '../../generated/resolvers-types'
 
 export const democracySummary = async (
-  parent: { address?: string },
-  args: { address?: string },
+  _: { address?: string },
```

Comment:
If these are not used let's change them to something like `Record<string, never>`, that `address` creates confusion

Reply:
Fixed here https://github.com/litentry/api-gateway/pull/77/commits/cff81e31d5d3ddc7261ce3a571ba7c3c8f7883ab